src/widget/wlibrarysidebar.cpp

```diff
@@ -93,19 +89,17 @@ void WLibrarySidebar::dragMoveEvent(QDragMoveEvent * event) {
             if (sidebarModel) {
                 accepted = false;
                 for (const QUrl& url : urls) {
-#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
```

Comment:
We have another policy, that says "backwards compatibility hacks for older dependencies should be removed only if new Mixxx features are incompatible with those old dependencies.": https://github.com/mixxxdj/mixxx/wiki/Minimum-requirements-policy

Even if I doubt, that main can be build with Qt5, we've to restore this, to comply with the policy.

Reply:
Okay, I reverted those specific changes. I also opened #13826 (which I actually did because of a separate review thread, before I was aware of your comment) to discuss whether it might make sense to reevaluate Qt5 vs. Qt6 when 4 years have passed since the last discussion.